compiler/damlc/lib/DA/Cli/Damlc.hs

```diff
@@ -303,6 +304,14 @@ cmdMergeDars =
   where
     cmd = execMergeDars <$> inputDarOpt <*> inputDarOpt <*> targetFileNameOpt
 
+cmdGenerateSrc :: Mod CommandFields Command
+cmdGenerateSrc =
+    command "generate-src" $
+    info (helper <*> cmd) $
+    progDesc "Generate DAML source code from a dalf package" <> fullDesc
+  where
+    cmd = execGenerateSrc <$> inputDalfOpt
```

Comment:
Should we add something like `-o output_directory`?

Reply:
let's keep it simple for now, I expect you create directory where you put your dalf into and run this.